src/ext/java/org/opentripplanner/ext/vectortiles/VectorTilesResource.java

```diff
@@ -41,43 +43,67 @@ public class VectorTilesResource {
   private static final Map<LayerType, LayerBuilderFactory> layers = new HashMap<>();
   private final OtpServerRequestContext serverContext;
   private final String ignoreRouterId;
+  private final Locale locale;
 
   static {
-    layers.put(LayerType.Stop, StopsLayerBuilder::new);
-    layers.put(LayerType.Station, StationsLayerBuilder::new);
+    layers.put(
+      LayerType.Stop,
+      (graph, transitService, layerParameters, locale) ->
+        new StopsLayerBuilder(transitService, layerParameters, locale)
+    );
+    layers.put(
+      LayerType.Station,
+      (graph, transitService, layerParameters, locale) ->
+        new StationsLayerBuilder(transitService, layerParameters, locale)
+    );
     layers.put(
       LayerType.VehicleRental,
-      (graph, transitService, layerParameters) ->
-        new VehicleRentalPlacesLayerBuilder(graph.getVehicleRentalStationService(), layerParameters)
+      (graph, transitService, layerParameters, locale) ->
+        new VehicleRentalPlacesLayerBuilder(
+          graph.getVehicleRentalStationService(),
+          layerParameters,
+          locale
+        )
     );
     layers.put(
       LayerType.VehicleRentalStation,
-      (graph, transitService, layerParameters) ->
+      (graph, transitService, layerParameters, locale) ->
         new VehicleRentalStationsLayerBuilder(
           graph.getVehicleRentalStationService(),
-          layerParameters
+          layerParameters,
+          locale
         )
     );
     layers.put(
       LayerType.VehicleRentalVehicle,
-      (graph, transitService, layerParameters) ->
+      (graph, transitService, layerParameters, locale) ->
         new VehicleRentalVehiclesLayerBuilder(
           graph.getVehicleRentalStationService(),
           layerParameters
         )
     );
-    layers.put(LayerType.VehicleParking, VehicleParkingsLayerBuilder::new);
-    layers.put(LayerType.VehicleParkingGroup, VehicleParkingGroupsLayerBuilder::new);
+    layers.put(
+      LayerType.VehicleParking,
+      (graph, transitService, layerParameters, locale) ->
+        new VehicleParkingsLayerBuilder(graph, layerParameters, locale)
+    );
+    layers.put(
+      LayerType.VehicleParkingGroup,
+      (graph, transitService, layerParameters, locale) ->
+        new VehicleParkingGroupsLayerBuilder(graph, layerParameters, locale)
+    );
   }
 
   public VectorTilesResource(
     @Context OtpServerRequestContext serverContext,
+    @Context Request grizzlyRequest,
     /**
      * @deprecated The support for multiple routers are removed from OTP2.
      * See https://github.com/opentripplanner/OpenTripPlanner/issues/2760
      */
     @Deprecated @PathParam("ignoreRouterId") String ignoreRouterId
   ) {
+    this.locale = grizzlyRequest.getLocale();
```

Comment:
Could we put this to `OtpServerRequestContext` in a method like `Locale requestLocale();`?